swap_meet/clothing.py

```diff
@@ -1,2 +1,15 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+
+    def __init__(self, id=None, fabric="Unknown", condition=0):
+        super().__init__(id, condition)
+        self.fabric = fabric
```

Comment:
Solid implementation of inheritance! Good job putting only immutable objects as default parameters here as well. 🎉